xbmc/cores/VideoPlayer/Edl.cpp

```diff
@@ -134,6 +134,11 @@ bool CEdl::ReadEdl(const std::string& strMovie, const float fFramesPerSecond)
       strFields[1] = strFields[0];
     }
 
+    if (StringUtils::StartsWith(strFields[0],"##"))
+    {
+      CLog::Log(LOGDEBUG, "Skipping comment line in EDL file");
```

Comment:
Have had a suggestion that maybe some more info about the skipped line may be useful.

Maybe something like to just log the line number in the file? Just may assist debugging

>       CLog::Log(LOGDEBUG, "Skipping comment on line %i in EDL file: %s", iLine, CURL::GetRedacted(edlFilename).c_str());